src/EventListener/BeforeRegistrationStartsListener.php

```diff
@@ -39,7 +40,7 @@ public function __invoke(BeforeRegistrationStartsEvent $event): void
                 'max_redirects' => 0,
             ]);
         } catch (\Throwable $e) {
```

Comment:
Can't we just catch the ones we want to ignore here?

```suggestion
        } catch (TransportExceptionInterface|RedirectionException $e) {
```